infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -382,8 +382,8 @@ public void adjustingP2pLowerBoundToUpperBound(final int p2pUpperBound) {
   }
 
   public void adjustingP2pUpperBoundToLowerBound(final int p2pLowerBound) {
-    log.info(
-        "Adjusting target number of peers upper bound to equal lower bound, which is {}",
+    log.warn(
+        "Target number of peers upper bound cannot be set below the default lower bound.  Increasing target to {}.",
```

Comment:
```suggestion
        "Target number of peers upper bound cannot be set below the peers lower bound.  Increasing target to {}.",
```

Reply:
The very slight difference I'm suggesting is that it's going to increase to the lower bound, whether you've set one or it's the default, so if you've overridden the lower bound it'll use your suggested setting...

Because of this removing 'default' is probably less misleading :) hope that makes sense...

Reply:
I think it do make sense